frontend/model/contracts/chatroom.js

```diff
@@ -520,6 +521,105 @@ sbp('chelonia/defineContract', {
       sideEffect ({ contractID, hash }) {
         emitMessageEvent({ contractID, hash })
       }
+    },
+    'gi.contracts/chatroom/voteOnPoll': {
+      validate: objectOf({
+        hash: string,
+        votes: arrayOf(string)
+      }),
+      process ({ data, meta, hash, id }, { state }) {
```

Comment:
Need to create messages only when `onlyRenderMessages` is `true`. Please add the follow lines just below the `process` function just like `addMessage/process` function.
```
if (!state.onlyRenderMessage) {
  return
}
```